app/static.go

```diff
@@ -11,15 +11,15 @@ import (
 	"time"
```

Comment:
I'd leave this file out of this change.  Just push this separately without a review.